### PR TITLE
getlantern/lantern#2166 add press section

### PR DESCRIPTION
**Pull request by fffw**
font, space and colour may not be accurate, but it works across Chrome/Firefox/IE/Mobile Chrome in various window width.

Also hided bitcoin donation icon (commented html code)

### CODE REVIEW

**Comment by myleshorton** [comment]
Awesome @fffw! Assigning to @atavism to review.

**Comment by atavism** [comment]
Awesome! I'm checking it out now. @myleshorton/@carsonefv, can you send me the PSD with the new press section to compare?

**Comment by fffw** [comment]
@atavism I have the PSD file but sorry I'm not clear what you email is?

**Comment by atavism** [comment]
Thanks, @fffw! Just sent you an e-mail.

File: app/index.html [hunk 1]
```diff
@@ -177,129 +177,158 @@ <h1 ng-bind-template="{{'EMAIL_SIGNUP' | translate}}">Sign Up For Our Newsletter
...
+        </a></li>
+        <li><a class="press-image"  target="_blank" href="http://www.lemonde.fr/pixels/article/2014/08/21/lantern-le-logiciel-qui-veut-eradiquer-la-censure-d-internet_4473868_4408996.html">
+          <img src="/images/press/le-monde.png" alt="Lantern, le logiciel qui veut éradiquer la censure d'Internet">
+        </a></li>
+      </ul>
+      <div class="praise">
+        <div class="user-praise-from-scmp" ng-bind-template="{{'USER_PRAISE_FROM_SCMP' | translate}}">&ldquo;Lantern is the most intuitive tool to access the internet freely&ddquo;</div>
+        <div class="user-scmp">
+        <span ng-bind-template="{{'USER_QUOTED' | translate}}"> - Lantern user quoted in the</span>
+        <a href="http://www.scmp.com/news/china-insider/article/1372661/use-lantern-software-means-view-banned-websites-grows-china" ng-bind-template="{{'SCMP' | translate}}">South China Morning Post</a>
```
Comment (atavism) [inline app/index.html]: I would add a target="_blank" attribute here so that the article opens in a new tab?

File: app/locale/en_US.json [hunk 2]
```diff
@@ -48,6 +48,9 @@
     "AN_APP_END_CENSORSHIP": "An app to end censorship",
     "THANKS_FOR_DOWNLOADING": "Thanks for downloading!",
     "END_CENSORSHIP_DESCRIPTION": "Internet censorship hurts billions, but we can stop it. Lantern is the app you install on your computer that lets the whole world escape censorship.",
+    "USER_PRAISE_FROM_SCMP": "“Lantern is the most intuitive tool to access the internet freely”",
+    "USER_QUOTED": "- Lantern user quoted in the",
+    "SCMP": "South China Morning Post",
```
Comment (atavism) [inline app/locale/en_US.json]: Awesome that you created translation strings for the press quotes already. I'll go ahead and push these to Transifex now.

File: app/index.html [hunk 3]
```diff
@@ -72,11 +72,11 @@
                     <li><a target="_blank" ng-href="{{FACEBOOK_URL}}"><span class="icon-facebook"></span></a></li>
                     <li><a target="_blank" ng-href="{{TUMBLR_URL}}"><span class="icon-tumblr"></span></a></li>
                     <li><a target="_blank" ng-href="{{GITHUB_URL}}"><span class="icon-github"></span></a></li>
-                    <li ng-click="trackEvent('donate')" class="bc"><div class="coinbase-button"
+                    <!--<li ng-click="trackEvent('donate')" class="bc"><div class="coinbase-button"
                         data-code="deeffa56195e84b5b9991ce7da426c9f"
                         data-button-style="none"></div><div
                         class="bc-link"><img
-                        src="/images/bitcoin-logo-cyan.png"></div></li>
+                        src="/images/bitcoin-logo-cyan.png"></div></li>-->
```
Comment (atavism) [inline app/index.html]: I actually figured out why this is broken, but I'll merge your changes and just uncomment the section before I deploy to production. Thanks!

File: app/index.html (hunk 1 above)
Comment (fffw) [inline app/index.html]: Oh you catch me!

**Comment by fffw** [comment]
Ok let me fix it and push again

**Comment by atavism** [comment]
Sweet! Merging.

**Pull request merged**